package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "xhtml-purifier",
-	"version": "0.3.1",
+	"version": "0.3.2",
```

Comment:
The way you modified ```purify``` makes it backwards-incompatible, right? because you added a new parameter that isn't the last parameter (and the default value is also not the old default value)